lib/matplotlib/backends/backend_qt4.py

```diff
@@ -1,48 +1,16 @@
 from __future__ import (absolute_import, division, print_function,
                         unicode_literals)
 
-import six
-from six import unichr
-import os
-import re
-import signal
-import sys
-
-import matplotlib
-
-from matplotlib.cbook import is_string_like
-from matplotlib.backend_bases import FigureManagerBase
 from matplotlib.backend_bases import FigureCanvasBase
-from matplotlib.backend_bases import NavigationToolbar2
-
-from matplotlib.backend_bases import cursors
-from matplotlib.backend_bases import TimerBase
-from matplotlib.backend_bases import ShowBase
-
-from matplotlib._pylab_helpers import Gcf
 from matplotlib.figure import Figure
 
+from .qt_compat import QtCore, QtWidgets, __version__
 
-from matplotlib.widgets import SubplotTool
-try:
-    import matplotlib.backends.qt_editor.figureoptions as figureoptions
-except ImportError:
-    figureoptions = None
-
-from .qt_compat import QtCore, QtWidgets, _getSaveFileName, __version__
-from matplotlib.backends.qt_editor.formsubplottool import UiSubplotTool
-
-from .backend_qt5 import (backend_version, SPECIAL_KEYS, SUPER, ALT, CTRL,
```

Comment:
Maybe that's the way to go--use `__all__` to declare the API.  There's a lot of stuff that we don't want to carry around forever.